### PR TITLE
Feature/mds 1686 continue refactoring mine resources

### PR DESCRIPTION
**Pull request by JustinMacaulay**
This is a refactor of the rest of the resources in the mine subfolder of mines. Mine commodity and mine disturbance were left out due the them needing a much larger re-work.  Compliance, Location, Region, tailings, and status all still require refactoring as well.

### CODE REVIEW

File: python-backend/app/api/mines/mine/resources/mine_type.py [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+    @api.marshal_with(MINE_TYPE_MODEL, code=201)
+    @api.doc(description='Creates a mine type and associates it with a Mine.')
     @requires_role_mine_create
     def post(self):
         data = self.parser.parse_args()
 
-        mine_guid = data['mine_guid']
-        mine_tenure_type_code = data['mine_tenure_type_code']
+        mine_guid = data.get('mine_guid')
+        mine_tenure_type_code = data.get('mine_tenure_type_code')
```
Comment (Jsyro) [inline python-backend/app/api/mines/mine/resources/mine_type.py]: these are required, i think the string indexing makes sense here.

Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine/resources/mine_type.py]: Changed.

**Comment by thegentlemanphysicist** [comment]
image
When a user attempts to download mine manger history I get the following error.

**Comment by JustinMacaulay** [comment]
As per a conversation with Aaron this morning the Download mine manager functionality was removed.

**Pull request closed without merging**